.tmp/tasks/2.2-system-settings-management.md

```
@@ -0,0 +1,509 @@
+  label: '新しい設定',
+  description: '設定の説明',
+  category: 'categoryName',
+  type: 'string',
+  defaultValue: '',
+  validation: { required: true },
+}
+```
+
+## ファイル構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)